src/components/panes/ActionPane.jsx

```diff
@@ -152,19 +165,95 @@ export default class ActionPane extends PaneBase {
                         />
                 );
             }
+            else {
+                responseList = (
+                    <div className="ActionPane-noResponses">
+                        <Msg id="panes.action.responses.none"/>
+                    </div>
+                );
+            }
 
             let contactSlot = this.props.connectContactDropTarget(
                 <div className="ActionPane-contactDropTarget">
                     <ContactSlot
                         contact={ action.contact }/>
+                        <Msg id="panes.action.contact.addContact"/>
                 </div>
             );
 
+            let reminderStatus = "checked";
+            let reminderStatusMsg = null;
+            let reminderButton = null;
+
+            if (participants &&
+                participants.find(p => !p.reminder_sent)) {
+                // Not all have reminders yet.
+                reminderStatus = "waiting";
+
+                let missingContact = null;
+
+                if (action.contact) {
+                    reminderButton = (
+                        <Button
+                            className="ActionPane-reminderButton"
+                            labelMsg="panes.action.reminders.button"
+                            onClick={ this.onClickReminders.bind(this) }/>
+                    );
+                }
+                else {
+                    missingContact = (
+                        <p className="missingContact">
+                            <Msg id="panes.action.reminders.missingContact"/>
+                        </p>
+                    );
+                }
+
+                reminderStatusMsg = (
+                    <div>
+                        <p className={ reminderStatus }>
+                            <Msg id="panes.action.reminders.received.notAll"/>
+                        </p>
+                        { missingContact }
+                    </div>
+                );
+            }
+            else {
+                // All have reminders
+                reminderStatusMsg = (
+                    <p className={ reminderStatus }>
+                        <Msg id="panes.action.reminders.received.all"/>
+                    </p>
+                );
+            }
+
+            let remindersActionBox = (
+                <ActionBox key="action"
+                    status={ reminderStatus }
+                    header="Reminders"
+                    content={ reminderStatusMsg }
```

Comment:
Setting children as properties is a little bit weird, but I appreciate the fact that you want several sets of children. Normally, this would be achieved using another solution, but given the time constraint I think we should let this be as is.